package.json

```diff
@@ -5,7 +5,8 @@
   "dependencies": {
     "react": "^16.12.0",
     "react-dom": "^16.12.0",
-    "react-scripts": "3.3.0"
+    "react-scripts": "3.3.0",
+    "styled-components": "^5.0.0"
```

Comment:
I personally enjoy using styled components because it gives me more control over how to style the component using a css in js library and I am able to leverage their usage of props to allow me to assert more control of styles based off my different states especially when dealing with alternating between Fahrenheit and Celsius. Additionally, it helps encapsulates certain styles within the component allowing me to know where in the codebase i need to go make changes in. Css modules also encapsulates the styles in the component and stylesheets allow for a more central place to write your styling, but I think the power of including JS for styling from styled-components, allow more power in creating a better application in regards to dealing with React's state.